joern-cli/frontends/x2cpg/src/main/scala/io/joern/x2cpg/utils/FreshNameGenerator.scala

```diff
@@ -1,4 +1,4 @@
-package io.joern.rubysrc2cpg.utils
+package io.joern.x2cpg.utils
```

Comment:
I do not think somethink like this is worth existing in x2cpg as a shared component. This is just an automatic `int + 1` and a function call.